.github/workflows/pr-commands.yml

```diff
@@ -0,0 +1,76 @@
+name: PR commands
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  entrypoint:
+    if: ${{ github.event.issue.pull_request }}
+    runs-on: ubuntu-latest
+    outputs:
+      command: ${{ steps.parse_command.outputs.command }}
+    steps:
+      - name: check auth
+        id: check_auth
+        run: |
+          # github handles
+          allowlist=(
+            "yihau"
+            "dmakarov"
+            "jcivlin"
+            "brson"
```

Comment:
only allow these people to trigger commands atm. we can add more later.

Reply:
Could you also add nvjle and ksolana, please?

Reply:
ah! my bad. sorry for miss you guys 😢 fixed: [c03e19c](https://github.com/solana-labs/move/pull/283/commits/c03e19cc403c09d59a47c3a24671aa9b66c90838)